engine/apps/api/views/alert_group.py

```diff
@@ -85,7 +85,11 @@ class AlertGroupFilter(DateRangeFilterMixin, ModelFieldFilterMixin, filters.Filt
     invitees_are = filters.ModelMultipleChoiceFilter(
         queryset=get_user_queryset, to_field_name="public_primary_key", method="filter_invitees_are"
     )
+    involved_users_are = filters.ModelMultipleChoiceFilter(
+        queryset=get_user_queryset, to_field_name="public_primary_key", method="filter_by_involved_users"
+    )
     with_resolution_note = filters.BooleanFilter(method="filter_with_resolution_note")
+    mine = filters.BooleanFilter(method="filter_mine")
```

Comment:
Alternative naming idea: "I'm involved" or something similar, seems to be easier to understand than "mine"

Reply:
Ack, makes sense. Will change in a next PR.